README.md

```diff
@@ -40,10 +40,11 @@
   * ### 1. Requirements
     * nodejs@4.4.3
     * npm@3 (`$ npm i -g npm`)
+    * yarn@1 (optional) (`$ curl -o- -L https://yarnpkg.com/install.sh | bash`)
```

Comment:
Should use `$ npm i -g yarn` to install it. Using curl can introduce a weird bug using HyperTerm, see:

* https://github.com/zeit/hyper/issues/858#issuecomment-253078634
* https://github.com/yarnpkg/yarn/issues/611#issuecomment-253157681